core/controllers/android_e2e_config.py

```diff
@@ -122,6 +123,18 @@ def post(self):
         topic.add_uncategorized_skill_id(skill_id)
         topic.add_subtopic(1, 'Test Subtopic Title')
 
+        # Save the dummy image to the filesystem to be used as thumbnail for
```

Comment:
@seanlip 
This is an earlier commit -- which had the changes for saving the thumbnail for subtopics. 
This change is not present in the latest commit. 
With the latest commit, this change is not present -- because we merged #13175 already.